packages/@react-aria/disclosure/src/useDisclosure.ts

```diff
@@ -114,6 +114,7 @@ export function useDisclosure(props: AriaDisclosureProps, state: DisclosureState
       // This can be overridden at the panel element level.
       role: 'group',
       'aria-labelledby': triggerId,
+      'aria-hidden': !state.isExpanded,
```

Comment:
> in the onKeyDown above, we'll want to check for e.repeat, otherwise holding down 'Enter' or 'Space' causes the accordion to open and close rapidly

Native details opens and closes repeatedly. Do we want to match that, or break away?

> also, there's no active state associated with the keydown, I think there should be?

I think we can add one to v3

Reply:
> Native details opens and closes repeatedly. Do we want to match that, or break away?

seems undesirable, but feel free to bring up with the team

Reply:
Went ahead and implemented it, but I'll bring it up with the team for opinions

Reply:
Outcome was to update this to use `onPressStart`, which already checks repeat